src/numerics.rs

```diff
@@ -0,0 +1,1270 @@
+use winrt::foundation::numerics::{Matrix3x2, Matrix4x4, Vector2, Vector3, Vector4};
+
+// Vector2
```

Comment:
I know you moved from the proc_macro to this, but this seems like a perfect use for `macro_rules!`. I think that would make this much more maintainable.

Reply:
Oops, I didn't see your comments until after I completed the PR.

Reply:
Now that the code is not being generated through the `build`/`import` macros, it does make sense to compact the definitions with `macro_rules` where practical, but we should keep it as simple as possible i.e. it doesn't need to be driven by metadata.